packages/sdk/src/react/hooks.ts

```
@@ -89,205 +95,178 @@ export function useEntityId(...keys: BigNumberish[]): BigNumberish {
- * Subscribe to entity changes. This hook fetches initial data from torii and subscribe to each entity change. Use `useModel` to access your data.
+ * Subscribe to entity changes. This hook fetches initial data from torii and subscribes to each entity change.
+ * Use `useModel` to access your data.
  *
  * @param query ToriiQuery
  */
-export function useEventQuery<Schema extends SchemaType>(
+export function useEntityQuery<Schema extends SchemaType>(
     query: ToriiQueryBuilder<Schema>
 ) {
-    const { sdk, useDojoStore } = useDojoSDK();
+    const { sdk, useDojoStore } = useDojoSDK<() => any, Schema>();
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)